runtime/rasdump/dmpmap.c

```diff
@@ -479,6 +499,57 @@ mapDumpSwitches(J9JavaVM *vm, J9RASdumpOption agentOpts[], IDATA *agentNum)
 
 	return OMR_ERROR_NONE;
 }
+
+/**
+ * Master disable heap, java, snap, and system dumps on out of memory
+ * @param vm Java vm
```

Comment:
This parameter is unused: why is it included?

Reply:
True.  Removed in both cases.